openedx/core/djangoapps/programs/utils.py

```diff
@@ -383,9 +383,6 @@ def _attach_run_mode_course_image_url(self, run_mode):
     def _attach_run_mode_course_url(self, run_mode):
         run_mode['course_url'] = reverse('course_root', args=[self.course_key])
 
-    def _attach_run_mode_end_date(self, run_mode):
```

Comment:
@clintonb @rlucioni I came across this as I was yanking out the start_datetime_text() function.  I also couldn't find it in use anywhere, but I asked you guys to step in to make sure I'm not being reckless and/or stupid.